crates/js-sys/tests/wasm/Number.rs

```diff
@@ -1,9 +1,17 @@
 use std::f64::{INFINITY, NAN};
 
+use wasm_bindgen::JsCast;
 use wasm_bindgen::JsValue;
 use wasm_bindgen_test::*;
 use js_sys::*;
 
+#[wasm_bindgen_test]
+fn number_inheritance() {
+    let number: i32 = 10;
```

Comment:
This line should be:

```rust
let number = Number::new(&JsValue::from(10));
```

Note that if you open a JS console:

```
> 10 instanceof Number
false
> new Number(10) instanceof Number
true
```

JS has a distinction between `Number` objects and its number values, which is a bit tricky!